edi_oca/models/edi_backend.py

```diff
@@ -353,6 +364,13 @@ def _send_retryable_exceptions(self):
         # when dealing w/ internal or external systems or filesystems
         return (IOError, OSError)
 
+    def _send_should_retry(self, exchange_record):
+        return (
+            # Safety check not to retry indefinitely
+            exchange_record.create_date >= (fields.Datetime.now() - timedelta(days=1))
+            or self.env.context.get("_edi_send_break_on_error")
```

Comment:
why should retry when the ctx key is set? They key is there to let if fail directly.